ui/js/components/auth.js

```diff
@@ -89,9 +89,7 @@ treeherder.component("login", {
             // Ask the back-end if a user is logged in on page load
             if (ctrl.userCanLogin) {
                 ThUserModel.get().then(async function (currentUser) {
-                    const userSession = localStorage.getItem('userSession');
-
-                    if (currentUser.email && userSession) {
+                    if (currentUser.email) {
                         ctrl.setLoggedIn(currentUser);
```

Comment:
`thNotify` stores recently generated notifications in localstorage, that can be viewed via the menu here:
https://screenshots.firefox.com/kJYZJGLsUHXkuksi/treeherder.mozilla.org

Perhaps adding notifications for certain auth events would (a) make it easier for the user to see what's going on at the time, (b) mean that when someone files a bug, we can ask them to check the notifications menu?